pulsar-broker/src/test/java/org/apache/pulsar/client/impl/BrokerClientIntegrationTest.java

```diff
@@ -365,6 +365,7 @@ public void testUnsupportedBatchMessageConsumer(SubscriptionType subType) throws
         // subscrie consumer2 with supporting batch version
         pulsarClient = PulsarClient.create(brokerUrl.toString());
         Consumer consumer2 = pulsarClient.subscribe(topicName, subscriptionName, conf);
+        Thread.sleep(pulsar.getConfiguration().getReadOnActiveConsumerDelayTimeMillis() + 500);
```

Comment:
same here? can we avoid `Thread.sleep()` because depending on it gives uncertain test-result and increases the test-time as well?